frontend/src/components/CreatePostModal.jsx

```diff
@@ -178,7 +178,7 @@ class CreatePostModal extends Component {
 	}
 
 	validPayload(requestBody) {
-		if (!(requestBody.title && requestBody.description && requestBody.content && requestBody.categories.length > 0)) {
+		if (!(requestBody.title && requestBody.description && requestBody.content)) {
```

Comment:
our backend still needs categories to work, only displaying categories is optional